common/djangoapps/third_party_auth/signals/tests/test_handlers.py

```diff
@@ -115,3 +144,112 @@ def test_saml_config_signal_handlers(
                     f"Expected '{error_message}' in error message for {description}, "
                     f"got: {error_calls[0][1][1]}"
                 )
+
+    def _get_current_provider(self, slug):
+        """
+        Helper to get current version of provider by slug.
+        """
+        return SAMLProviderConfig.objects.current_set().get(slug=slug)
+
+    def _get_site(self, site_id):
+        """
+        Helper to get site by ID (1 = site1, 2 = site2).
+        """
+        return self.site1 if site_id == 1 else self.site2
+
+    def _create_new_config(self, site, slug='default'):
+        """
+        Helper to create new SAML config and trigger signal.
+        """
+        return SAMLConfigurationFactory(
+            site=site,
+            slug=slug,
+            entity_id=f'https://{site.domain}/updated'
+        )
+
+    @ddt.data(
+        # Args: provider_site_id, provider_slug, signal_saml_site_id, signal_saml_slug, is_provider_updated
```

Comment:
```suggestion
        # Args: provider_site_id, provider_slug, signal_saml_site_id, signal_saml_slug, should_provider_update
```

Is this more apt ? 

Or maybe `will_provider_update` ?

Reply:
Both are fine. Its just which will sounds more clear.

Reply:
Or, `provider_update_expected`? I'm good with any option, include leaving as-is. I'll leave it up to you two to decide.